src/main/java/kubatech/api/utils/ItemUtils.java

```diff
@@ -0,0 +1,35 @@
+package kubatech.api.utils;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+
+import kubatech.mixin.mixins.minecraft.ItemStackAccessor;
+
+public class ItemUtils {
+
+    public static NBTTagCompound writeItemStackToNBT(ItemStack stack) {
+        NBTTagCompound compound = new NBTTagCompound();
+
+        compound.setInteger("id", Item.getIdFromItem(((ItemStackAccessor) (Object) stack).getItem()));
+        compound.setInteger("Count", stack.stackSize);
```

Comment:
I assume the purpose of this method is for saving stack size larger than `byte`. Why not call `ItemStack#writeToNBT` and then overwrite `Count` tag with integer? That way itemstack accessor can be removed, and potentially it works with modifications made by other mods.

Reply:
Yeah, simplified